example/vectorAdd/src/main.cpp

```diff
@@ -73,21 +73,41 @@ class VectorAddKernel
     }
 };
 
+template<class A, class B>
+#if defined(ALPAKA_ACC_CPU_BT_OMP4_ENABLED)
+using DefaulAcc = alpaka::acc::AccCpuOmp4<A,B>;
+#elif defined(ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED)
+using DefaulAcc = alpaka::acc::AccCpuOmp2Blocks<A,B>;
+#elif defined(ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLED)
+using DefaulAcc = alpaka::acc::AccCpuFibers<A,B>;
+#elif defined(ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLED)
+using DefaulAcc = alpaka::acc::AccCpuOmp2Threads<A,B>;
+#elif defined(ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED)
+using DefaulAcc = alpaka::acc::AccCpuSerial<A,B>;
+#elif defined(ALPAKA_ACC_CPU_B_SEQ_T_THREADS_ENABLED)
+using DefaulAcc = alpaka::acc::AccCpuThreads<A,B>;
+#elif defined(ALPAKA_ACC_GPU_CUDA_ENABLED)
+using DefaulAcc = alpaka::acc::AccGpuCudaRt<A,B>;
+#else
+#define NOP
+#warning "No supported backend selected."
```

Comment:
It looks like this branch would not compile, as there is nothing after `template<class A, class B>`

Reply:
Fixed.